src/pages/faq.js

```diff
@@ -0,0 +1,196 @@
+import React from "react"
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const FAQ = () => {
+  return (
+    <Layout>
+      <SEO title="FAQ" />
+      <div className="container faq">
+        <h1 id="main" className="py-2">
+          Frequently Asked Questions
+        </h1>
+
+        <p>
+          Do you have questions? We've compiled some of the most common ones we
+          get into this one page.
+        </p>
+
+        <h2>Heat Pumps</h2>
+
+        <h3>Do heat pumps work in colder climates?</h3>
+
+        <p>
+          Google this question or ask a contractor and you might think the
+          answer is no. But make no mistake, heat pumps absolutely work in cold
+          climates. Not only that, but often they are the most energy efficient
+          and cost effective solution available. A few decades ago, most heat
+          pumps stopped working when the temperature dropped below 20 or 30
+          degrees fahrenheit. By contrast, today’s heat pumps can run more
+          efficiently than any other HVAC system all the way down to about -25
+          fahrenheit. Just ask the millions of homeowners in Scandinavia. People
+          in Norway, Finland and Sweden are installing heat pumps at a faster
+          pace than anywhere else in Europe.
+          <a href="https://carbonswitch.com/do-heat-pumps-work-in-cold-weather/">
+            Read more at CarbonSwitch
+          </a>
```

Comment:
I think we should also add:

"In colder climates, a potential hurdle to installing heat pumps is home insulation. You may want to get a [blower door test](https://www.energy.gov/energysaver/blower-door-tests) to determine how much heat you are leaking from your home"